sites/offshore-mag/server/templates/index.marko

```diff
@@ -34,26 +34,111 @@ $ const adSlots = {
+            limit=3
+            section-alias="drilling-completion"
+            header={ title: "Drilling & Completion", href: "/drilling-completion" }
+            native-x={ placement: 'list1', aliases: ['drilling-completion'], index: 3 }
+          />
+        </div>
+      </div>
+      <div class="row">
+        <div class="col-lg-6 mb-block">
+          <endeavor-content-query-section-list
+            limit=3
+            section-alias="field-development"
+            header={ title: "Field Development", href: "/field-development" }
+            native-x={ placement: 'list1', aliases: ['field-development'], index: 2 }
+          />
+        </div>
+        <div class="col-lg-6 mb-block">
+          <endeavor-content-query-section-list
+            limit=3
+            section-alias="subsea"
+            header={ title: "Subsea", href: "/subsea" }
+            native-x={ placement: 'list1', aliases: ['subsea'], index: 2 }
+          />
+        </div>
+      </div>
+
     </div>
     <div class="col-lg-4 ad-rail">
       <endeavor-gam-ad-unit-display id="gpt-ad-rail1" />
       <endeavor-gam-ad-unit-display id="gpt-ad-rail2" />
     </div>
   </div>
 
+  <div class="row">
+    <div class="col-lg-4 mb-block">
+      <endeavor-content-query-section-list
+        limit=3
+        section-alias="geosciences"
+        header={ title: "Geosciences", href: "/geosciences" }
+        native-x={ placement: 'list1', aliases: ['geosciences'], index: 2 }
+      />
+    </div>
+    <div class="col-lg-4 mb-block">
+      <endeavor-content-query-section-list
+        limit=3
+        section-alias="pipelines"
+        header={ title: "Pipelines", href: "/pipelines" }
+        native-x={ placement: 'list1', aliases: ['pipelines'], index: 2 }
+      />
+    </div>
+    <div class="col-lg-4 mb-block">
+      <endeavor-content-query-section-list
+        limit=3
+        section-alias="production"
+        header={ title: "Production", href: "/production" }
+        native-x={ placement: 'list1', aliases: ['production'], index: 2 }
+      />
+    </div>
+  </div>
+
+  <div class="row">
+    <div class="col-lg-4 mb-block">
+      <endeavor-content-query-section-list
+        limit=3
+        section-alias="rigs-vessels"
+        header={ title: "Rigs/Vessels", href: "/rigs-vessels" }
+        native-x={ placement: 'list1', aliases: ['rigs-vessels'], index: 2 }
+      />
+    </div>
+    <div class="col-lg-4 mb-block">
+      <endeavor-content-query-section-list
+        limit=3
+        section-alias="deepwater"
+        header={ title: "Deepwater", href: "/deepwater" }
+        native-x={ placement: 'list1', aliases: ['deepwater'], index: 2 }
+      />
+    </div>
+    <div class="col-lg-4 mb-block">
+      <endeavor-content-query-section-list
+        limit=3
+        section-alias="business-briefs"
+        header={ title: "Business Briefs", href: "/business-briefs" }
+        native-x={ placement: 'list1', aliases: ['business-briefs'], index: 2 }
```

Comment:
This will fallback to default because `business-briefs` is not a valid NativeX topic (per instructions yesterday)